### PR TITLE
Data update

### PR DESCRIPTION
**Pull request by cancelei**
Hello reviewer,

Here are the required steps to complete this step of the Veterinarian Clinic Database:
**Step 1: Populating**
image

**Step 2: Deleting and Rolling back:**
**2.1: Deleting**
image

**2.2 Rolling Back**
image

**Step 3:**
**-- 3.1: Delete all animals born after Jan 1st, 2022. + Create a savepoint for the transaction.**
image

**-- 3.2: Update all animals' weight to be their weight multiplied by -1 then Rollback to the savepoint.**
_Printscreen before rollback._
image

**-- 3.3 Update all animals' weights that are negative to be their weight multiplied by -1 then Commit transaction.**
image

**Step 4** Answer questions below with printscreen of the database:
**4.1 -- How many animals are there?**
image

**4.2 -- How many animals have never tried to escape?**
image

**4.3 -- What is the average weight of animals?**
image

**4.4 -- Who escapes the most, neutered or not neutered animals?** Neutered escaped more.
image

**4.5 -- What is the minimum and maximum weight of each type of animal?**
image

**4.6 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?**
image

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -3,6 +3,7 @@
 CREATE TABLE animals (
     id serial PRIMARY KEY,
     name varchar(100),
+    spicies varchar(100),
```
Comment (mikethreels) [inline schema.sql]: - [ ] it is not good practice to change an existing table instead you're supposed to use the alter table query.

File: queries.sql [hunk 2]
```diff
@@ -21,3 +21,67 @@ SELECT * FROM animals WHERE name != 'Gabumon';
...
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- TRANSACTIONS:
+
+-- T1: Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```
Comment (mikethreels) [inline queries.sql:31]: - [ ] according to the requirements you should make sure the change was undone as well after ROLLBACK

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Glauber, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job creating the queries.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @cancelei 

Good job implementing the changes suggested by the previous reviewer 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ Table is updated
-  ✅ Used correct Gitflow
-  ✅ Screenshots added

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**